### ISSUE TITLE
about BYOL performance

### ISSUE THREAD

**Issue by jisunkwo** [open-mmlab/mmselfsup#89]
Hi, 
Where can I find the config file for the byol model, byol_r50-e3b0c442.pth, in the Model Zoo??
I couldn't find 'selfsup/byol/r50_bs4096_ep200.py ' from this git repository.
Also I fine-tuned on object detection task on mmdetection with [faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py](https://github.com/open-mmlab/mmdetection/blob/master/configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py) and the bbox_mAP is only '**27.00**' which is too low compared with '[40.3](https://github.com/open-mmlab/OpenSelfSup/blob/master/docs/MODEL_ZOO.md#coco2017-object-detection)' in your benchmark page.
Even though the architecture and some hyper-parms are differ from yours, the difference is too big.
With imagenet-pretrained(pytorch) model, the mAP reporduced as 37.2 and with my personal byol model it shows 36.3.
Do you have any idea about this accuracy drop with your byol model??
I trained it on 4 gpus and 8 samples per gpu so that I increased the learning rate to 0.04(x2) as guided in [the mmdetection document page](https://github.com/open-mmlab/mmdetection/blob/master/docs/1_exist_data_model.md#train-predefined-models-on-standard-datasets).

**Comment by XiaohangZhan** [comment]
This experiment was removed since the result is not as expected due to some bugs previously. We did not try on mmdet. Please make sure all the backbone params are corrected loaded.

**Comment by jisunkwo** [comment]
Every backbone params are loaded correctly. You mean the uploaded model I used is not expected to show high accuracy?? Let me know if you think the mAP 27.0 is reasonable on faster rcnn 1x. Or do you recommend me to use another byol model such as byol_r50_bs256_accmulate16_ep300-5df46722.pth or byol_r50_bs2048_accmulate2_ep200-e3b0c442.pth??

**Comment by XiaohangZhan** [comment]
We did not try mmdet, so we cannot guarantee it works on mmdet. r50_bs4096_ep200 can achieve 40.3 if you test it with our standard detection setting, i.e., using the detectron framework following `getting_started.md`. In r50_bs4096_ep200, we did not use SyncBN, which might affect the performance. So we remove it. Please use r50_bs2048_accumulate2_ep200_fp16.

**Comment by jisunkwo** [comment]
got it. Thanks for your answer. :)
